Shared/BetterColorPicker.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using BepInEx;
+using BepInEx.Configuration;
+using ChaCustom;
+using HarmonyLib;
+using Illusion.Component.UI.ColorPicker;
+using IllusionUtility.GetUtility;
+using KKAPI.Utilities;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+using PluginInfo = BetterColorPicker.Info;
+
+namespace BetterColorPicker
+{
+    [BepInDependency(KKAPI.KoikatuAPI.GUID)]
```

Comment:
Version should always be added to avoid cryptic crashes.

Reply:
Added the version information.